db/migrate/20210416094652_add_validated_to_documents.rb

```diff
@@ -0,0 +1,6 @@
+class AddValidToDocuments < ActiveRecord::Migration[6.1]
+  def change
+    add_column :documents, :validated, :boolean, required: true, default: true
```

Comment:
Have we confirmed that this defaults to true? I realise we didn't capture this in the acceptance criteria, but I think it's an important product question.

Option 1) as has currently been built where new documents are valid by default
Option 2) documents are invalid by default (I suspect this would be weird, but wanted to capture it!)
Option 3) documents by default have a null validation status, where they are neither invalid or valid until they are marked as so.

I'll ping this comment to Ali.

Reply:
Option 3. Documents should be neither valid or invalid until the planning officer has marked it as so!